app/src/org/commcare/logging/AndroidLogSerializer.java

```diff
@@ -78,11 +79,12 @@ public void writeToDeviceReport(XmlSerializer serializer) throws IOException {
         serializer.startTag(DeviceReportWriter.XMLNS, "log_subreport");
 
         try {
-            if (isSingleLog) {
-                serializeLog(entry);
-            } else {
-                if (Logger._() != null) {
-                    Logger._().serializeLogs(this);

```

Comment:
Seems like `serializeLogs` is not called anymore. Maybe you can remove the implementations from `AndroidLogger` then?